demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -51,7 +51,9 @@ def rotate_poses(poses_3d, R, t):
                       type=str, required=True)
     args.add_argument('-i', '--input',
                       help='Required. Path to input image, images, video file or camera id.',
-                      nargs='+', default='')
+                      required=True)
+    args.add_argument('-loop', '--loop', default=False, action='store_true',
+                      help='Optional. Run the input data in a loop.')
```

Comment:
Align help messages with C++. Take object_detection_demo_yolov3_async as an example, but keep --input for python version as alternative to -i.

Don't forget to update `README.md` after.

Reply:
done

Reply:
You should refer to VideoCapture as `cv2.VideoCapture` in python.

Reply:
Why parentheses after cv2.VideoCapture?

Reply:
I deleted it